code/modules/antagonists/wraith/critters/abilities/plaguerat.dm

```diff
@@ -149,6 +149,9 @@ ABSTRACT_TYPE(/datum/targetable/critter/plague_rat)
 		if (BOUNDS_DIST(holder.owner, target) > 0)
 			boutput(holder.owner, "<span class='alert'>That is too far away to bite.</span>")
 			return TRUE
+		if (holder.owner.hasStatus("disposals")) //No biting in Disposals, that's cheating.
+			boutput(holder.owner, "<span class='alert'>You definitely can't bite through pipes.</span>")
+			return TRUE
```

Comment:
Should probably be a more general check for "are we standing on a turf" - probably don't want them biting out of (closed) closets etc either.